lib/elixir/lib/list.ex

```diff
@@ -577,6 +577,51 @@ defmodule List do
   end
 
   @doc """
+  If `enum` starts with `prefix`, return true. Otherwise, return false.
```

Comment:
We should change the tense from imperative to present simple.
We also need to add backticks. And the structures of summaries is usually in the form of: "Returns true if _condition_; otherwise returns false."

IMO it should read:

    Returns `true` if `list` starts with `prefix`; otherwise returns `false`.